### ISSUE TITLE
Issue with score() Method in MatrixProfile Implementation (TSB_UAD/models/matrix_profile.py)

### ISSUE THREAD

**Issue by chenyuanTKCY** [TheDatumOrg/TSB-UAD#7]
The following code snippet represents the process:
'''
modelName='MatrixProfile'
clf = MatrixProfile(window = slidingWindow)
x = data
clf.fit(x)
clf.score(query_length=2*slidingWindow,dataset=x)
score = clf.score
score = MinMaxScaler(feature_range=(0,1)).fit_transform(score.reshape(-1,1)).ravel()
score = np.array([score[0]]*math.ceil((slidingWindow-1)/2) + list(score) + [score[-1]]*((slidingWindow-1)//2))
plotFig(data, label, score, slidingWindow, fileName=name, modelName=modelName) #, plotRange=[1775,2200]
'''
However, it seems that the current implementation of the MatrixProfile class does not include a score() method. I would like to set up my experiment in the same way as yours. Could you please clarify how the score() method should be implemented or provide any additional details necessary for consistency?
